src/main/java/bio/terra/app/configuration/ApplicationConfiguration.java

```diff
@@ -444,4 +448,29 @@ public WebServerFactoryCustomizer<TomcatServletWebServerFactory> tomcatCustomize
             connector ->
                 connector.setEncodedSolidusHandling(EncodedSolidusHandling.DECODE.getValue()));
   }
+
+  @Bean("azureIPs")
+  public Map<Integer, List<String>> azureIPs() throws IOException {
+    URL url =
+        new URL(
+            "https://download.microsoft.com/download/7/1/D/71D86715-5596-4529-9B13-DA13A5DE5B63/ServiceTags_Public_20240708.json");
```

Comment:
I just hard-coded this url for testing btw. It actually changes when the file is updated once a week so our options are to:
1. Parse the download url from [https://www.microsoft.com/en-us/download/details.aspx?id=56519](url)
2. Use the [https://learn.microsoft.com/en-us/rest/api/virtualnetwork/service-tags/list?view=rest-virtualnetwork-2024-01-01&tabs=HTTP](url) API endpoint (this requires a subscription id though)

Reply:
ah dang yeah thats annoying, i think using the API endpoint is the move, not sure about the subscription we should use for that though, id ask about it in #dsp-azure

Reply:
The second object won't get use Azure IP ranges. It'll just list the Public IPs within a subscription, which are a specific Azure resource